src/components/Character/Character.jsx

```diff
@@ -55,28 +61,35 @@ export default function Character(props){
 			aria-roledescription="slide"
 			aria-label={`Character ${index + 1} of ${characterData.length}.`}
 			aria-hidden={ hidden }>
-			<article>
-				<h1 id={`character__${id}`}>
-					{ name }
-				</h1>
-				<h2>
-					{ nickname }
-				</h2>
-				<h3>
-					{ actor }
-				</h3>
-				<p>
-					{ body }
-				</p>
+			<article className={ s.container }>
+				<div className={ s.content }>
+					<div className={ s.text }> 
+						<h1 className={ s.name } 
+							id={`character__${id}`}
+						>
+							{ name }
+						</h1>
+						<h2 className={ s.nickName } >
```

Comment:
can we make this `nickname` to match the variable being used underneath - seems weird to have multiple naming conventions happening here.